src/Compilers/VisualBasic/Portable/Syntax/Syntax.xml

```diff
@@ -4985,7 +4985,7 @@ Unify Enum declarations with other block type declarations.
         <native-equiv name="Expression.Opcodes.Nothing" />
       </node-kind>
 
-      <child name="Token" kind="IntegerLiteralToken|CharacterLiteralToken|DecimalLiteralToken|FloatingLiteralToken|DateLiteralToken|StringLiteralToken|TrueKeyword|FalseKeyword|NothingKeyword">
+      <child name="Token" kind="IntegerLiteralToken|CharacterLiteralToken|DecimalLiteralToken|FloatingLiteralToken|DateLiteralToken|StringLiteralToken|DollarSignDoubleQuoteToken|DoubleQuoteToken|InterpolatedStringTextToken|TrueKeyword|FalseKeyword|NothingKeyword">
```

Comment:
This change was made because VisualBasicSyntaxFactsService defines `IsLiteral` as `Return SyntaxFacts.IsLiteralExpressionToken(CType(token.Kind, SyntaxKind))` as opposed to CSharpSyntaxFactsService::IsLiteral which explicitly lists token kinds in a switch statement.